x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -180,6 +181,15 @@ function initializeExecutionStatus(
   };
 }
 
+function isEmptyObject(obj: {}) {
+  for (const attr in obj) {
+    if (obj.hasOwnProperty(attr)) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
nit: Do we need to do `return JSON.stringify(obj) === JSON.stringify({})`?

ref: https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object

Reply:
if you scroll down ;)
![image](https://user-images.githubusercontent.com/189600/111381442-1ae1bb80-867c-11eb-8c3e-89584168842c.png)

Reply:
haha, I stop reading after I see the accepted answer 🤦